reference/UiTPAS.v2.json

```diff
@@ -3053,6 +3833,212 @@
         "x-tags": [
           "Models"
         ]
+      },
+      "Reward": {
+        "type": "object",
+        "title": "Reward",
+        "description": "",
+        "x-tags": [
+          "Models"
+        ],
+        "properties": {
+          "id": {
+            "type": "string",
+            "readOnly": true
+          },
+          "title": {
+            "type": "string"
+          },
+          "owningCardSystem": {
+            "$ref": "#/components/schemas/CardSystem"
+          },
+          "applicableCardSystems": {
+            "type": "array",
+            "items": {
+              "$ref": "#/components/schemas/CardSystem"
+            }
+          },
+          "organizers": {
+            "type": "array",
+            "items": {
+              "$ref": "#/components/schemas/Organizer"
+            }
+          },
+          "allCardSystems": {
+            "type": "boolean"
+          },
+          "status": {
+            "type": "string",
+            "enum": [
+              "ACTIVE",
+              "INACTIVE",
+              "DELETED"
+            ],
+            "readOnly": true
+          },
+          "maxAvailableUnits": {
+            "type": "integer"
+          },
+          "unitsTaken": {
+            "type": "integer",
+            "readOnly": true
+          },
+          "description1": {
+            "type": "string"
+          },
+          "description2": {
```

Comment:
Can we rename these to something that better explains what each one is used for? I personally have no clue 😅